cmd/image-processing/main_test.go

```diff
@@ -157,34 +180,70 @@ var _ = Describe("Image Processing Resource", func() {
 		})
 
 		It("should fail in case mandatory arguments are missing", func() {
-			Expect(run()).To(HaveOccurred())
+			Expect(run()).ToNot(Succeed())
 		})
 
 		It("should fail in case --image is empty", func() {
-			Expect(run("--image", "")).To(HaveOccurred())
+			Expect(run(
+				"--image", "",
+			)).To(FailWith("argument must not be empty"))
 		})
 
 		It("should fail in case --image does not exist", func() {
 			Expect(run(
-				"--image", "docker.io/feqlQoDIHc/bcfHFHHXYF",
-			)).To(HaveOccurred())
+				"--image", "docker.io/library/feqlqodihc:bcfhfhhxyf",
+			)).To(FailWith("unexpected status code 401"))
 		})
 
 		It("should fail in case annotation is invalid", func() {
 			withTestImage(func(tag name.Tag) {
 				Expect(run(
+					"--insecure",
```

Comment:
This was a bug in the test cases actually, the test case did not test what was intended to test.